Controls/LoadablePanel/src/DotVVM.Contrib.Samples/Controls/Pager/Pager.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using DotVVM.Framework.Binding;
+using DotVVM.Framework.Compilation.Javascript;
+using DotVVM.Framework.Compilation.Javascript.Ast;
+using DotVVM.Framework.Controls;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DotVVM.Contrib.Samples.Controls.Pager
+{
+    public static class PagerExtensions
+    {
+        public static void GoToPage(IPageableGridViewDataSet set, int index, Func<IPageableGridViewDataSet> loader)
+        {
+            throw new NotImplementedException("JS");
+        }
+
+        public static IServiceCollection AddPagerExtensions(this IServiceCollection services)
+        {
+            services.Configure((Action<JavascriptTranslatorConfiguration>)(c =>
+            {
+                RegisterAlertMethod(c, nameof(PagerExtensions.GoToPage), "goToPage");
+            }));
+            return services;
+        }
+
+        private static void RegisterAlertMethod(JavascriptTranslatorConfiguration c, string csharpName, string jsName)
```

Comment:
I don't get the name of this method, can you elaborate?

Reply:
Oh, forgot to rename that one...